src/fairseq2/recipes/lm/_online_finetune/_online_dpo.py

```diff
@@ -83,7 +81,7 @@ class OnlineDpoFinetuneUnit(TrainUnit[SequenceBatch]):
 
     _reference_model: Module | RemoteVllmModel | None
     _vllm_model: RemoteVllmModel
-    _vllm_actors: Dict[str, RemoteVllmModel]
+    _vllm_actors: Dict[str, Union[RemoteVllmModel, RemoteHFModel]]
```

Comment:
should we change the names of these. maybe something like `_rollout_model` and `_ray_actors`?

Reply:
yes, I like this

Reply:
I'm going to do this in another PR because I think it's a breaking change in old configs that should now be `sync_vllm_model_every_n_steps` --> `sync_rollout_model_every_n_steps`